tests/vllm/test_qaic_output_consistency.py

```diff
@@ -0,0 +1,142 @@
+# -----------------------------------------------------------------------------
+#
+# Copyright (c) 2024 Qualcomm Innovation Center, Inc. All rights reserved.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# -----------------------------------------------------------------------------
+
+import gc
+import random
+
+import pytest
+from vllm import LLM, SamplingParams
+
+test_models = [
+    "TinyLlama/TinyLlama-1.1B-Chat-v1.0",
+]
+
+seq_len = 128
+ctx_len = 256
+decode_bsz = 4
+dtype = "mxfp6"
+kv_dtype = "mxint8"
+device_group = [0]
+
+
+@pytest.mark.parametrize("model_name", test_models)
+def test_output_consistency(model_name):
+    sampling_params = SamplingParams(temperature=0.0, max_tokens=None)
+
```

Comment:
Please add proper docstring for each test to explain it better.

Reply:
Added docustring explaining the test.